evap/evaluation/models.py

```diff
@@ -1894,12 +1894,9 @@ def construct_mail(self, to_email, cc_addresses, subject_params, body_params):
         subject = self.render_string(self.subject, subject_params, autoescape=False)
         plain_content = self.render_string(self.plain_content, body_params, autoescape=False)
 
-        rendered_content = self.render_string(
-            self.html_content if self.html_content else self.plain_content.replace("\n", "<br />"), body_params
-        )
-        wrapper_template_params = dict(
-            {"email_content": rendered_content, "email_subject": escape(subject)}, **body_params
-        )
+        html_content = self.html_content if self.html_content else linebreaksbr(self.plain_content)
```

Comment:
@niklasmohrin: [`linebreaksbr`](https://github.com/django/django/blob/main/django/template/defaultfilters.py#L469) happens to do exactly what we need here (escape, then replace \n with <br/>). However, this is not guaranteed by the [documentation for this filter specifically](https://docs.djangoproject.com/en/4.0/ref/templates/builtins/#linebreaksbr). What do you think? We probably want a test for escaping here, anyway, and if we have that, we could keep `linebreaksbr`?

Reply:
If we have a test as our safety net, I don't have anything against `linebreaksbr` :+1: